### PR TITLE
Update to current gir version

### PR DESCRIPTION
**Pull request by sophie-h**

### CODE REVIEW

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
I was going to suggest updating the gir-files as well as they were updated for 3.99.3. But I guess that can happen later as we need to figure out the sys crate situation for gtk4 crates first.

File: src/auto/functions.rs [hunk 1]
```diff
@@ -198,21 +193,15 @@ pub fn print_run_page_setup_dialog<P: IsA<Window>>(
     }
 }
 
-pub fn print_run_page_setup_dialog_async<
-    P: IsA<Window>,
-    Q: FnOnce(&PageSetup) + Send + Sync + 'static,
```
Comment (sdroege) [inline src/auto/functions.rs:203]: Hrm, this looks wrong

Comment (sdroege) [reply src/auto/functions.rs:203]: See https://github.com/gtk-rs/gir/pull/981

Comment (sdroege) [reply src/auto/functions.rs:203]: If you update gir another time, it should be all good now :)

Comment (sophie-h) [reply src/auto/functions.rs]: Done

**Review by sdroege — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**